app/models/school.rb

```diff
@@ -85,6 +85,9 @@ class School < ApplicationRecord
   has_many :mentor_memberships
   has_many :mentors, through: :mentor_memberships
 
+  normalizes :email_address, with: ->(value) { value.strip.downcase }
+  validates :email_address, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_nil: true
```

Comment:
Let's move this to line 94, after the `validates :name` validation. This keeps validations all together.